install/homebrew.md

```diff
@@ -9,12 +9,19 @@ This installation method is designed for easy install and upgrades, and uses san
 
 ##### Prerequisites
 
-In order to install and Airsonic on macOS all you need is [Homebrew](https://brew.sh).
+In order to install and Airsonic on macOS you need Java 8 runtime and [Homebrew](https://brew.sh).
 
-Airsonic requires a recent Java runtime. The formula requires this cask, but it will not be installed automatically.
+Download the _JDK 8 .dmg_ package from the [JDK download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) ([direct download link](http://download.oracle.com/otn-pub/java/jdk/8u191-b12/2787e4a523244c269598db4e85c51e0c/jdk-8u191-macosx-x64.dmg)).
 
-```
-brew cask install java
+Install the downloaded package.
+
+Add run the following lines in your terminal:
+
+```bash
+# Use ~/.zshrc if you're using zsh.
+
+echo export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)" >> ~/.bash_profile \
+source ~/.bash_profile
```

Comment:
Is this the best way of doing this ? I don't use macOS at all so I hope this is the most common and clean way to do this.

Reply:
so you can manually add the `export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"` to your .bashrc/.bash_profile/.zshrc. source reloads the bash/zsh instead of restarting the terminal. the other thing is usually when you install java, you don't even need to do this... i could add to the script to check if java 8 is there, if not, add to the bash profile.